src/scene/forward-renderer.js

```diff
@@ -516,6 +513,13 @@ class ForwardRenderer {
         this._createAreaLightPlaceholderLuts();
     }
 
+    // Static properties used by the Profiler in the Editor's Launch Page
+    static skipRenderCamera = null;
```

Comment:
should these defines also be placed in `#if PROFILER` ?

Reply:
it's probably simpler to leave them out so the editor can just access them with any engine build.

Reply:
Done

Reply:
Oh. Just saw @mvaligursky disagreement with your suggestion @slimbuck. I went ahead and made them PROFILER build only. TBH, I don't think it's a big deal. This implementation is actually pretty nasty for a number of reasons - rather than refactor, I wanted to do the minimum to repair the original functionality. But in general:

1. There shouldn't be Editor-specific functionality in the engine. I'd rather define a general-purpose public API (with better naming!) for limiting draws calls, or early-outing from rendering a layer composition.
2. These properties are static and would break if two `Application` instances are active on a page.
3. There seems to be layer-specific draw call limiting code in there too and I'm not sure if it's currently being used. Needs investigation.